src/quo2/components/text_input.cljs

```diff
@@ -0,0 +1,270 @@
+(ns quo2.components.text-input
+  (:require [clojure.spec.alpha :as s]
+            [reagent.core :as reagent]
```

Comment:
Are we modifying text-input file for make changes required for search input? If we implement search-input for quo2 component, probably these changes are not required.

Reply:
we don't use new-ui flag. anymore, so if we don't implement new component in this PR, just use old component without changes in it , so we know we need to implement it

Reply:
cool